src/__tests__/form-test.tsx

```diff
@@ -78,6 +78,29 @@ test('custom validator', async () => {
     );
 });
 
+test('when there are multiple errors on submit, the fields with error are anounced by screen reader', async () => {
```

Comment:
what happens when multiple fields contain errors? All errors gets announced?

If so, can you add a test for this case?